### PR TITLE
Update 0200-12-30-introduction.md

### PR DESCRIPTION
**Pull request by kateregga1**
Translating Introduction unit

### CODE REVIEW

File: _posts/sw/0200-12-30-introduction.md [hunk 1]
```diff
@@ -8,71 +8,35 @@ category: beginner
...
 
 ![A village in Indonesia][]
 
-Maps are visual symbols of our world. They can often 
-demonstrate an idea better than words. This in turn can help us
-answer important questions. Where is the closest school or hospital? Who
-has the least access to these facilities? Where is poverty the most
-problematic? Questions like these can often best be expressed with maps,
-and maps can help find solutions to these questions. 
+RAMANI ni picha ya dunia yetu yenye maelezo lukuki. inaweza elezea wazo flani vizuri kuliko maneno. Hii kwa upande wetu inaweza kusaidia kujibu maswali muhimu. Ni shule au hospitali gani ipo karibu? Nani aliye karibu na huduma hizo? wapi umaskini ni tatizo zaidi? maswali kama haya yanaweza elezewa vizuri kwa kutumia ramani, vilevile ramani zinaweza tusaidia kutafuta ufumbuzi juu ya matatizo hayo.
```
Comment (althio) [inline _posts/sw/0200-12-30-introduction.md]: "RAMANI" is all caps.
Should it be "Ramani"?
Can you modify it or should we keep it like that?

File: _posts/sw/0200-12-30-introduction.md [hunk 2]
```diff
@@ -8,71 +8,35 @@ category: beginner
 
 This section of LearnOSM is in the process of being translated. If you would like to assist with translating this site, please see [CONTRIBUTING.md](https://github.com/hotosm/learnosm/blob/gh-pages/CONTRIBUTING.md).
```
Comment (althio) [inline _posts/sw/0200-12-30-introduction.md]: You can remove this once the page is translated.

File: _posts/sw/0200-12-30-introduction.md [hunk 3]
```diff
@@ -8,71 +8,35 @@ category: beginner
...
 ============
 
-Information is powerful. With good information and the right
-understanding, individuals and communities are better able to improve
-their lives and make good decisions about the future. There are many
-people and organizations who make decisions that affect our lives. Good
-information allows these NGOs, governments and citizens to make better
-decisions, and hopefully make our lives better. Maps are a good way
-to convey information. 
+Taarifa zina nguvu. Kwa taarifa nzuri na uelewa wa kweli, watu na jamii wanaweza kuboresha maisha yao na kufanya maamuzi mazuri ya baadae. Kuna watu wengi na mashirika yanayofanya maamuzi yanayoathiri  maisha yetu.Taarifa nzuri zinawezesha mashirika binafsi,serikali na raia kufanya maamuzi mazuri na  matumaini ya kuboresha maisha yetu.Ramani ni njia nzuri ya kusambaza habari.
```
Comment (althio) [inline _posts/sw/0200-12-30-introduction.md]: yetu.Taarifa
Needs a space.

**Comment by althio** [comment]
Hi @kateregga1 
Very good work, thank you!
It looks well done.

I don't know how to write Kiswahili, but could you double-check if they are some typos like no space after a full stop or comma, no capital letters at beginning of sentence, double-space...

File: _posts/sw/0200-12-30-introduction.md (hunk 1 above)
Comment (kateregga1) [inline _posts/sw/0200-12-30-introduction.md]: Yes it should be Ramani, thanks

File: _posts/sw/0200-12-30-introduction.md (hunk 2 above)
Comment (kateregga1) [inline _posts/sw/0200-12-30-introduction.md]: I have removed the typos, do I need to change the images to ones from Tanzania?

Comment (althio) [inline _posts/sw/0200-12-30-introduction.md]: > do I need to change the images to ones from Tanzania?

No you don't have to.
We usually keep these images for all languages in this page.

File: _posts/sw/0200-12-30-introduction.md [hunk 4]
```diff
@@ -8,71 +8,35 @@ category: beginner
...
 
 ![A village in Indonesia][]
 
-Maps are visual symbols of our world. They can often 
-demonstrate an idea better than words. This in turn can help us
-answer important questions. Where is the closest school or hospital? Who
-has the least access to these facilities? Where is poverty the most
-problematic? Questions like these can often best be expressed with maps,
-and maps can help find solutions to these questions. 
+Ramani ni picha ya dunia yetu yenye maelezo lukuki. Inaweza elezea wazo flani vizuri kuliko maneno. Hii kwa upande wetu inaweza kusaidia kujibu maswali muhimu. Ni shule au hospitali gani ipo karibu? Nani aliye karibu na huduma hizo? Wapi umasikini ni tatizo zaidi? maswali kama haya yanaweza elezewa vizuri kwa kutumia ramani, vilevile ramani zinaweza tusaidia kutafuta ufumbuzi juu ya matatizo hayo.
```
Comment (althio) [inline _posts/sw/0200-12-30-introduction.md]: > zaidi? maswali

Needs capital?

**Comment by althio** [comment]
I think we are almost ready to merge.
Thank you for your hard work!

**Comment by althio** [comment]
Merging, we should remove the header soon.

**Comment by kateregga1** [comment]
Thanks Althio

How is the procedure, can I pull another request now?

**Comment by althio** [comment]
Yes you can open another pull request for more translated files.

**Pull request merged**